### DISCUSSION TITLE
Rectangle with radius or not in the nicegui.io

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1551] [General]
As an official site ,If such thing is normal.  If they should both with radius or not 

![微信截图_20230904152517](https://github.com/zauberzeug/nicegui/assets/25096045/717f6de4-623a-4946-8bbf-08fb4bfde546)

**Comment by rodja** [comment]
I converted this to an issue: https://github.com/zauberzeug/nicegui/issues/1552
